apps/desktop-wallet/src/components/SnackbarManager.tsx

```diff
@@ -56,14 +56,10 @@ const SnackbarPopup = memo(({ message }: { message: Required<SnackbarMessage> })
 
   // Remove snackbar popup after its duration
   useEffect(() => {
-    let timer: ReturnType<typeof setTimeout>
-
     if (message && message.duration >= 0) {
-      timer = setTimeout(() => dispatch(snackbarDisplayTimeExpired()), message.duration)
-    }
+      const timer = setTimeout(() => dispatch(snackbarDisplayTimeExpired()), message.duration)
```

Comment:
We sometimes use the custom `sleep` function from web3 (which is basically a handle for `setTimeout`), but we usually use `setTimeout` directly. 
The sleep function can be a bit confusing IMO because it feels like it could stop the whole execution (like in other languages). But I like how concise it is compared to `setTimeout`.
However the most important point:
Wit `sleep` we can't clear the timeout like we do here. This may be an issue. WDYT?